.github/workflows/lint.yml

```diff
@@ -24,28 +24,25 @@ jobs:
       - name: Running pre-commit
         uses: pre-commit/action@2c7b3805fd2a0fd8c1884dcaebf91fc102a13ecd # v3.0.1
 
-  buildifier:
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Checking out repository
-        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
-      - name: Fetching Base Branch
-        # We have to explicitly fetch the base branch as well
-        run: git fetch --no-tags --prune --depth=1 origin "${GITHUB_BASE_REF?}:${GITHUB_BASE_REF?}"
+      # Run these hooks manually until they work as expected on all platforms.
       - name: Downloading Buildifier
         run: |
           mkdir -p "${HOME?}/bin"
           wget https://github.com/bazelbuild/buildtools/releases/download/4.0.1/buildifier -O "${HOME?}/bin/buildifier"
           chmod +x "${HOME?}/bin/buildifier"
           echo "${HOME?}/bin" >> $GITHUB_PATH
-      - name: Running buildifier on changed files
-        run: |
-          # Fail if the script fails (unfixable lint warnings) or it creates a
-          # diff (fixable lint warnings or formatting).
-          EXIT_CODE=0
-          ./build_tools/scripts/run_buildifier.sh "${GITHUB_BASE_REF?}" || EXIT_CODE=1
-          git diff --exit-code
-          exit "${EXIT_CODE?}"
+      - name: Running pre-commit - bazel-buildifier
+        uses: pre-commit/action@2c7b3805fd2a0fd8c1884dcaebf91fc102a13ecd # v3.0.1
+        with:
+          extra_args: --hook-stage manual --all-files bazel-buildifier
```

Comment:
Dunno why this needs `--all-files`. The `files` here should be triggering for our `BUILD.bazel` files: https://github.com/jlebar/pre-commit-hooks/blob/f2d115a052860b09b2888b4f104be614bf3b4779/.pre-commit-hooks.yaml#L1-L6